src/Workspaces/Core/Portable/TemporaryStorage/TemporaryStorageServiceFactory.cs

```diff
@@ -65,33 +66,24 @@ internal sealed partial class TemporaryStorageService : ITemporaryStorageService
 
     /// <summary>
     /// The most recent memory mapped file for creating multiple storage units. It will be used via bump-pointer
-    /// allocation until space is no longer available in it.
+    /// allocation until space is no longer available in it.  Access should be synchronized on <see cref="_gate"/>
     /// </summary>
-    /// <remarks>
-    /// <para>Access should be synchronized on <see cref="_gate"/>.</para>
-    /// </remarks>
     private ReferenceCountedDisposable<MemoryMappedFile>.WeakReference _weakFileReference;
 
-    /// <summary>The name of the current memory mapped file for multiple storage units.</summary>
-    /// <remarks>
-    /// <para>Access should be synchronized on <see cref="_gate"/>.</para>
-    /// </remarks>
+    /// <summary>The name of the current memory mapped file for multiple storage units. Access should be synchronized on
+    /// <see cref="_gate"/></summary>
```

Comment:
📝 The access condition is not part of the description of this field. The old form of this comment was preferred.